assets/js/refCtrl.js

```diff
@@ -375,27 +237,10 @@ module.exports = function ($scope, $filter) {
   };
 
   $scope.setLocalBan = function (ban) {
-    var url = "/mod/setlocalban";
-    io.socket.post(url, {userId: $scope.refUser.id, ban: ban}, function (data, res) {
-      if (res.statusCode === 200) {
-        $scope.refUser.banned = data.banned;
-        $scope.$apply();
-      } else {
-        console.log("Error");
-      }
-    });
+    return sharedService.setLocalBan($scope, $scope.refUser.name, ban);
```

Comment:
One of them is for the ban/unban button on user profiles, and one of them is for the unban button on the banlist.

That said, I'm not sure we need the one on the banlist; I'mean occasionally worried that I'll accidentally hit the wrong button and unban someone.